projects/packages/my-jetpack/src/products/class-backup.php

```diff
@@ -184,7 +184,8 @@ private static function get_state_from_wpcom() {
 		$response = Client::wpcom_json_api_request_as_blog( sprintf( '/sites/%d/rewind', $site_id ) . '?force=wpcom', '2', array( 'timeout' => 2 ), null, 'wpcom' );
 
 		if ( 200 !== wp_remote_retrieve_response_code( $response ) ) {
-			return new WP_Error( 'rewind_state_fetch_failed' );
+			$status = new WP_Error( 'rewind_state_fetch_failed' );
```

Comment:
Just out of curiosity, I'm wondering why the error is assigned to a variable now before being returned?

Reply:
Spoke too soon, got my answer here: pdWQjU-Sf-p2- thanks for writing that up!

Reply:
Thanks for taking a look!
I see now that it's not clear from the code, and GitHub hides the part where the variable is declared static.
I should've left a note in the PR as well.